packages/types/src/Score.ts

```diff
@@ -40,11 +51,14 @@ export interface IScoreAggregateDetails {
     score: number
   }
 }
+
 export interface IScoreAverageDetails {
```

Comment:
We don't need this at the SDK level. This computation should be done at the API level